src/webextension/manage/components/item-details.js

```diff
@@ -14,9 +14,12 @@ import Toolbar from "../../widgets/toolbar";
 
 import styles from "./item-details.css";
 
+import * as telemetry from "../../telemetry";
+
 const PASSWORD_DOT = "\u25cf";
 
 function CopyToClipboardButton({text, ...props}) {
+  telemetry.recordEvent("itemCopied", "entryDetails");
```

Comment:
This is going to fire the "itemCopied" event every time the button is drawn on the page, not when actually clicked.